cla_backend/apps/call_centre/serializers.py

```diff
@@ -342,6 +344,12 @@ class CreateCaseSerializer(CaseSerializer):
     No other fields can be used when creating a case atm.
     """
 
+    def create(self, validated_data):
+        validated_data['gtm_anon_id'] = str(uuid.uuid4())
+
+        instance = Case.objects.create(**validated_data)
```

Comment:
Is it possible to call super here? Otherwise the super class will not have it's create method called

Reply:
Added call to super